### PR TITLE
Feat/add endpoints

### PR DESCRIPTION
**Pull request by Ramimashkouk**

### CODE REVIEW

File: backend/df_designer/app/cli.py [hunk 1]
```diff
@@ -11,6 +12,23 @@
 
 cli = typer.Typer()
 
+
+def _execute_command(command_to_run):
+    try:
+        process = subprocess.run(command_to_run.split(),check=False)
```
Comment (RLKRo) [inline backend/df_designer/app/cli.py:18]: Maybe these should be wrapped in `df_designer.app:BuildProcess` to access it's api (and for consistency)?

File: backend/df_designer/app/clients/process_manager.py [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+    async def start(self, process_type: str, cmd_to_run: str):
+        id_ = randint(1, 10000) #TODO: change it to incremental counter
+        if process_type == "build":
+            process = BuildProcess(id_, _get_preset_end_status(cmd_to_run))
+        elif process_type == "run":
+            process = RunProcess(id_, _get_preset_end_status(cmd_to_run))
+        else: raise ValueError("process_type should be one of [build, run]")
         await process.start(cmd_to_run)
-        self.processes[process.pid] = process
+        self.processes[id_] = process
```
Comment (RLKRo) [inline backend/df_designer/app/clients/process_manager.py:28]: Would be nice to return process id here.

File: backend/df_designer/pyproject.toml [hunk 3]
```diff
@@ -9,11 +9,13 @@ packages = [{include = "app"}]
...
 python = "^3.10"
 fastapi = "^0.110.0"
-uvicorn = "^0.27.1"
+uvicorn = {extras = ["standard"], version = "^0.28.0"}
 pydantic = "^2.6.3"
 typer = "^0.9.0"
 pydantic-settings = "^2.2.1"
 aiofiles = "^23.2.1"
+cookiecutter = "^2.6.0"
+dff = {extras = ["telegram"], version = "^0.7.0"}
```
Comment (RLKRo) [inline backend/df_designer/pyproject.toml:18]: Why telegram?

File: backend/df_designer/app/clients/process_manager.py (hunk 2 above)
Comment (Ramimashkouk) [reply backend/df_designer/app/clients/process_manager.py:28]: Resolved in 53cc260341758d90355b79dac606772c81d939f2

File: backend/df_designer/pyproject.toml (hunk 3 above)
Comment (Ramimashkouk) [reply backend/df_designer/pyproject.toml:18]: had no idea back then. resolved in later commit

**Pull request merged**